app/controllers/katello/api/v2/module_streams_controller.rb

```diff
@@ -4,15 +4,26 @@ class Api::V2::ModuleStreamsController < Api::V2::ApiController
     apipie_concern_subst(:a_resource => N_("a module stream"), :resource => "module_streams")
     include Katello::Concerns::Api::V2::RepositoryContentController
 
-    before_action :check_hosts, :only => :index
+    before_action :check_params, :only => :index
 
     update_api(:index) do
       param :host_ids, Array, :desc => N_("List of host id to list available module streams for")
+      param :name_stream_only, :boolean, :desc => N_("Return name and stream information only)")
```

Comment:
I'm not seeing `name_stream_only` or `host_ids` show up in the api docs even when I run apipie:cache and restart the server. I'm on the page `apidoc/v2/module_streams.html`. Let me know if you see it there and I'm missing something, otherwise it may be a bug in apipie